src/main/resources/earlierversions/detect8.sh

```diff
@@ -38,7 +68,7 @@ DETECT_SOURCE=${DETECT_SOURCE:-}
 # *that* location will be used.
 # *NOTE* We currently do not support spaces in the
 # DETECT_JAR_DOWNLOAD_DIR.
-DEFAULT_DETECT_JAR_DOWNLOAD_DIR="${HOME}$(get_path_separator)synopsys-detect$(get_path_separator)download"
+DEFAULT_DETECT_JAR_DOWNLOAD_DIR="${HOME}$(get_path_separator)$(get_detect_name_prefix)$(get_path_separator)download"
```

Comment:
@dmamidisynopsys I added the conditional logic for "detect" vs "synopsys-detect" to this line as well.
Seems like this won't affect execution and is more of a branding decision.
If users use detect9.sh or detect8.sh, the jar download would take place in a directory named "synopsys-detect". Let me know in case we want to just hard code it to "detect" always.

Reply:
@niravrsynopsys No this is good. We should keep the old names for 8 and 9. I am assuming we are doing this for .ps1s too. It should be consistent

Reply:
@devmehtabd had confirmed this case doesn't exist in the powershell scripts.
I checked and I see the default detect jar download path in powershell seems to have always been an empty string. https://github.com/synopsys-sig/synopsys-detect-scripts/blob/5f2a6202c16dd590ee79ed946692643d9af0e2be/src/main/resources/templates/detect-ps.ps1#L40

Reply:
Oh wow uff! Looks like it is simply using /tmp there. Thanks for clarifying.